tests/test_ops_unary.py

```diff
@@ -128,7 +128,6 @@ def test_clip(min_value, max_value):
     assert np.allclose(ng_results, [expected])
 
 
-@pytest.mark.xfail(reason='Workaround for bug in nGraph. https://github.com/NervanaSystems/ngraph/issues/1196')
 @pytest.mark.parametrize('min_value, max_value', [
     (0., np.finfo(np.float32).max),
```

Comment:
We can move this arguments from `test_clip_broken` to `test_clip` and remove `test_clip_broken` completely.